### PR TITLE
Fix titles for GCP Policy packs

### PR DESCRIPTION
**Pull request by madhushreeray30**

### CODE REVIEW

File: policy_packs/gcp/computeengine/enforce_disks_to_be_attached_to_instances/main.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 resource "turbot_policy_pack" "main" {
-  title       = "Enforce GCP Compute Engine Disks to Be Attached to Instances"
+  title       = "Enforce GCP Compute Engine Disks Are Attached to Instances"
   description = "Ensure that all allocated storage is actively used and monitored, reducing the risk of unnecessary expenses and potential security vulnerabilities associated with unattached disks."
   akas        = ["gcp_computeengine_enforce_disks_to_be_attached_to_instances"]
```
Comment (SumitPopat) [inline policy_packs/gcp/computeengine/enforce_disks_to_be_attached_to_instances/main.tf:4]:
```suggestion
  akas        = ["gcp_computeengine_enforce_disks_are_attached_to_instances"]
```

File: policy_packs/gcp/computeengine/enforce_disks_to_not_be_older_than_7_days/main.tf [hunk 2]
```diff
@@ -1,5 +1,5 @@
 resource "turbot_policy_pack" "main" {
-  title       = "Enforce GCP Compute Engine Disks to Not Be Older Than 7 Days"
+  title       = "Enforce GCP Compute Engine Disks Are Not Older Than 7 Days"
   description = "Enforcing disks to not be older than 7 days is critical to ensure that data storage is continuously refreshed and aligned with the latest security and performance standards."
   akas        = ["gcp_computeengine_enforce_disks_to_not_be_older_than_7_days"]
```
Comment (SumitPopat) [inline policy_packs/gcp/computeengine/enforce_disks_to_not_be_older_than_7_days/main.tf:4]:
```suggestion
  akas        = ["gcp_computeengine_enforce_disks_are_not_older_than_7_days"]
```

File: policy_packs/gcp/computeengine/enforce_instances_to_not_be_older_than_7_days/main.tf [hunk 3]
```diff
@@ -1,5 +1,5 @@
 resource "turbot_policy_pack" "main" {
-  title       = "Enforce GCP Compute Engine Instances to Not Be Older Than 7 Days"
+  title       = "Enforce GCP Compute Engine Instances Are Not Older Than 7 Days"
   description = "Enforcing instances to not be older than 7 days is important to ensure that instances are regularly updated and patched, minimizing the risk of vulnerabilities and security exploits."
   akas        = ["gcp_computeengine_enforce_instances_to_not_be_older_than_7_days"]
```
Comment (SumitPopat) [inline policy_packs/gcp/computeengine/enforce_instances_to_not_be_older_than_7_days/main.tf:4]:
```suggestion
  akas        = ["gcp_computeengine_enforce_instances_are_not_older_than_7_days"]
```

File: policy_packs/gcp/computeengine/enforce_instances_to_not_use_external_ip_address/main.tf [hunk 4]
```diff
@@ -1,5 +1,5 @@
 resource "turbot_policy_pack" "main" {
-  title       = "Enforce GCP Compute Engine Instances to Not Use External IP Addresses"
+  title       = "Enforce GCP Compute Engine Instances Do Not Use External IP Addresses"
   description = "Limiting instances to internal IP addresses only minimizes exposure to the internet, thereby protecting sensitive data and systems from unauthorized access and potential threats."
   akas        = ["gcp_computeengine_enforce_instances_to_not_use_external_ip_address"]
```
Comment (SumitPopat) [inline policy_packs/gcp/computeengine/enforce_instances_to_not_use_external_ip_address/main.tf:4]:
```suggestion
  akas        = ["gcp_computeengine_enforce_instances_do_not_use_external_ip_address"]
```

File: policy_packs/gcp/computeengine/enforce_instances_to_not_use_specific_machine_types/main.tf [hunk 5]
```diff
@@ -1,5 +1,5 @@
 resource "turbot_policy_pack" "main" {
-  title       = "Enforce GCP Compute Engine Instances to Not Use Specific Machine Types"
+  title       = "Enforce GCP Compute Engine Instances Do Not Use Specific Machine Types"
   description = "Enforce instances to not use specific machine types helps prevent the use of machine types that may be unsuitable for certain workloads, excessively costly, or lacking necessary security features, thereby optimizing resource utilization and maintaining a secure environment."
   akas        = ["gcp_computeengine_enforce_instances_to_not_use_specific_machine_types"]
```
Comment (SumitPopat) [inline policy_packs/gcp/computeengine/enforce_instances_to_not_use_specific_machine_types/main.tf:4]:
```suggestion
  akas        = ["gcp_computeengine_enforce_instances_do_not_use_specific_machine_types"]
```

File: policy_packs/gcp/computeengine/enforce_instances_to_not_use_unapproved_images/main.tf [hunk 6]
```diff
@@ -1,5 +1,5 @@
 resource "turbot_policy_pack" "main" {
-  title       = "Enforce GCP Compute Engine Instances to Not Use Unapproved Images"
+  title       = "Enforce GCP Compute Engine Instances Use Only Approved Images"
   description = "Ensure that all instances adhere to organizational standards, reducing the risk of vulnerabilities."
   akas        = ["gcp_computeengine_enforce_instances_to_not_use_unapproved_images"]
```
Comment (SumitPopat) [inline policy_packs/gcp/computeengine/enforce_instances_to_not_use_unapproved_images/main.tf:4]:
```suggestion
  akas        = ["gcp_computeengine_enforce_instances_use_only_approved_images"]
```

File: policy_packs/gcp/iam/enforce_user_service_accounts_to_not_have_admin_privileges/main.tf [hunk 7]
```diff
@@ -1,5 +1,5 @@
 resource "turbot_policy_pack" "main" {
-  title       = "Enforce GCP IAM User-Managed Service Accounts to Not Have Admin Privileges"
+  title       = "Enforce GCP IAM User-Managed Service Accounts Do Not Have Admin Privileges"
   description = "Minimize the risk of unauthorized access and potential misuse of administrative capabilities."
   akas        = ["gcp_iam_enforce_user_service_accounts_to_not_have_admin_privileges"]
```
Comment (SumitPopat) [inline policy_packs/gcp/iam/enforce_user_service_accounts_to_not_have_admin_privileges/main.tf:4]:
```suggestion
  akas        = ["gcp_iam_enforce_user_service_accounts_do_not_have_admin_privileges"]
```

File: policy_packs/gcp/kms/check_crypto_keys_are_rotated_regularly/README.md [hunk 8]
```diff
@@ -3,7 +3,7 @@ categories: ["data protection", "security"]
 primary_category: "data protection"
 ---
 
-# Enforce GCP KMS Crypto Keys to be rotated on regular basis
+# Check If GCP KMS Crypto Keys Are Rotated Regularly
```
Comment (SumitPopat) [inline policy_packs/gcp/kms/check_crypto_keys_are_rotated_regularly/README.md:6]: Pleaes check if the Policy Pack AKA matches the title here.

File: policy_packs/gcp/network/check_https_is_enforced_for_load_balancers/main.tf [hunk 9]
```diff
@@ -1,5 +1,5 @@
 resource "turbot_policy_pack" "main" {
-  title       = "Check If GCP Network Load Balancers Enforce HTTPS to Manage Encrypted Web Traffic"
+  title       = "Check GCP Network Load Balancers Enforce HTTPS for Encrypted Web Traffic"
   description = "Ensure that the data transmitted between clients and load-balanced applications is encrypted, protecting it from interception and unauthorized access."
   akas        = ["gcp_network_check_https_is_enforced_for_load_balancers"]
```
Comment (SumitPopat) [inline policy_packs/gcp/network/check_https_is_enforced_for_load_balancers/main.tf:4]:
```suggestion
  akas        = ["gcp_network_check_load_balancers_enforce_https"]
```

File: policy_packs/gcp/network/enforce_default_vpc_network_is_not_used_for_projects/README.md [hunk 10]
```diff
@@ -3,7 +3,7 @@ categories: ["networking", "security"]
 primary_category: "networking"
 ---
 
-# Enforce Default VPC Network to Not Be Used Within GCP Projects
+# Enforce Default VPC Network Is Not Used in GCP Projects
```
Comment (SumitPopat) [inline policy_packs/gcp/network/enforce_default_vpc_network_is_not_used_for_projects/README.md:6]: Please check if the Policy Pack AKA matches the title.

File: policy_packs/gcp/network/enforce_firewall_rules_with_port_ranges_to_not_allow_incoming_traffic/main.tf [hunk 11]
```diff
@@ -1,5 +1,5 @@
 resource "turbot_policy_pack" "main" {
-  title       = "Enforce GCP VPC Network Firewall Rules with Port Ranges to Not Allow Incoming Traffic"
+  title       = "Enforce GCP VPC Network Firewall Rules with Port Ranges to Block Incoming Traffic"
   description = "Ensure that only necessary and specific ports are open for inbound traffic, minimizing the risk of unauthorized access and potential attacks."
   akas        = ["gcp_network_enforce_firewall_rules_with_port_ranges_to_not_allow_incoming_traffic"]
```
Comment (SumitPopat) [inline policy_packs/gcp/network/enforce_firewall_rules_with_port_ranges_to_not_allow_incoming_traffic/main.tf:4]:
```suggestion
  akas        = ["gcp_network_enforce_firewall_rules_with_port_ranges_to_block_incoming_traffic"]
```

File: policy_packs/gcp/network/enforce_firewall_to_not_allow_egress_access/main.tf [hunk 12]
```diff
@@ -1,5 +1,5 @@
 resource "turbot_policy_pack" "main" {
-  title       = "Enforce GCP Network Firewall to Not Allow Any Egress Access"
+  title       = "Enforce GCP Network Firewall to Block All Egress Access"
   description = "Ensure that no outbound traffic is permitted, preventing data exfiltration and unauthorized communication with external systems."
   akas        = ["gcp_network_enforce_firewall_to_not_allow_egress_access"]
```
Comment (SumitPopat) [inline policy_packs/gcp/network/enforce_firewall_to_not_allow_egress_access/main.tf:4]:
```suggestion
  akas        = ["gcp_network_enforce_firewall_to_block_egress_access"]
```

File: policy_packs/gcp/storage/enforce_buckets_to_not_be_older_than_7_days/main.tf [hunk 13]
```diff
@@ -1,5 +1,5 @@
 resource "turbot_policy_pack" "main" {
-  title       = "Enforce GCP Storage Buckets to Not Be Older Than 7 Days"
+  title       = "Enforce GCP Storage Buckets Are Not Older Than 7 Days"
   description = "Ensure that only recently created buckets are in use, reducing the risk of using outdated configurations and improving data security and compliance with best practices"
   akas        = ["gcp_storage_enforce_buckets_to_not_be_older_than_7_days"]
```
Comment (SumitPopat) [inline policy_packs/gcp/storage/enforce_buckets_to_not_be_older_than_7_days/main.tf:4]:
```suggestion
  akas        = ["gcp_storage_enforce_buckets_are_not_older_than_7_days"]
```

File: policy_packs/gcp/kms/check_crypto_keys_are_rotated_regularly/README.md (hunk 8 above)
Comment (madhushreeray30) [reply policy_packs/gcp/kms/check_crypto_keys_are_rotated_regularly/README.md:6]: yes it matches > `gcp_kms_check_crypto_keys_are_rotated_regularly`

File: policy_packs/gcp/network/enforce_default_vpc_network_is_not_used_for_projects/README.md (hunk 10 above)
Comment (madhushreeray30) [reply policy_packs/gcp/network/enforce_default_vpc_network_is_not_used_for_projects/README.md:6]: Updated 👍

**Review by SumitPopat — APPROVED** [review]

**Pull request merged**